src/Panel/Panel/Panel.example.jsx

```diff
@@ -12,23 +12,24 @@ render(
     <div className="example-block">
       <span>Panels:</span>
 
-
       <Panel
         x={0}
         y={20}
         title="Collapsible"
         collapsible={true}
-      />
+      >
+        Content
+      </Panel>
 
       <Panel
         x={200}
         y={20}
-        closable={true}
         collapsible={true}
-        title="Tooltips"
-        compressTooltip="Einklappen"
-        closeTooltip="Schließen"
-      />
+        title="Tooltip"
+        collapseTooltip="Einklappen"
```

Comment:
Maybe be could only use english texts in `react-geo`to make texts / tooltips more consistently.

Reply:
As this is the explicit example for setting alternate tooltips i used german.
The default value is ofcourse english: `collapse`